MQTTnet.Core/Client/MqttClientQueued.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using MQTTnet.Core.Adapter;
+using MQTTnet.Core.Diagnostics;
+using MQTTnet.Core.Exceptions;
+using MQTTnet.Core.Packets;
+using MQTTnet.Core.Protocol;
+using MQTTnet.Core.Internal;
+
+namespace MQTTnet.Core.Client
+{
+    public class MqttClientQueued : MqttClient, IMqttClientQueued
```

Comment:
Please do not use inheritance. One clean code principle is "composition over inheritance".  As I said please use the client as a field.

Reply:
Ok you do not like hineritance. Gonna change.

Reply:
Done